package.json

```diff
@@ -8,15 +8,20 @@
     "@babel/core": "^7.21.8",
     "@babel/plugin-syntax-jsx": "^7.21.4",
```

Comment:
Remove caret ("^") and tilda ("~")symbols

Reply:
resolved